src/app/campaign/store/effects/campaign.effects.ts

```diff
@@ -83,6 +89,70 @@ export class CampaignEffects {
             deletedFlightDocs: { [id: number]: HalDoc };
             updatedFlightDocs: { [id: number]: HalDoc };
             createdFlightDocs: { [id: number]: HalDoc };
+          }) => {
+            const flightDaysDocs: Observable<{ [id: number]: HalDoc[] }> = createdFlightDocs
+              ? forkJoin(
+                  Object.values(createdFlightDocs).reduce(
+                    (acc, doc) => ({ ...acc, [doc.id]: this.campaignService.loadFlightDays(doc) }),
+                    {}
+                  )
+                )
+              : of({});
+            return flightDaysDocs.pipe(
+              map(createdFlightDaysDocs => ({
+                campaignDoc,
+                deletedFlightDocs,
+                updatedFlightDocs,
+                createdFlightDocs,
+                createdFlightDaysDocs
+              }))
+            );
+          }
+        ),
+        // retrieve the flights' days from each updated flight response
```

Comment:
Comments++!  (And some good observable-fu loading all the campaign -> flights -> flight-days in here!)

Reply:
positive reinforcement++ it works!